### PR TITLE
Update index-eng.html

### PR DESCRIPTION
**Pull request by ajay004**

### CODE REVIEW

File: index-eng.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
+    <meta charset = "UTF-8">
```
Comment (duboisp) [inline index-eng.html]: Can you remove the space beside the "=" character in the attribute
Can you use tabs for indentation.

**Review by duboisp — CHANGES_REQUESTED** [review]
Can you do the French also?

Thanks

**Comment by shawnthompson** [comment]
I created a working example of the PR.

https://servicecanada.github.io/wet-boew-jvanulde-geomap-improvements/demos/geomap/geomap-en.html

Can we test that the change works.

**Comment by LaurentGoderre** [comment]
This change it not really needed since those are just redirects. I wonder if we should remove them altogheter

**Comment by lucas-hay** [comment]
@LaurentGoderre Should we remove altogether or?

**Comment by LaurentGoderre** [comment]
I would remove them personally. We haven't used this file in over 2 years I think

**Pull request closed without merging**